__tests__/db.test.js

```diff
@@ -311,4 +311,22 @@ describe('PATCH /api/articles/:article_id', () =>{
                 expect(res.statusCode).toBe(404)
         })
     })
+})
+
+describe('DELETE /api/comments/:comment_id', ()=>{
+    test('Returns a 204 with no content after deleting comment', ()=>{
+        return request(app)
+        .delete('/api/comments/1')
+        .then((response)=>{
+            expect(response.statusCode).toBe(204)
+            expect(Object.keys(response.body).length).toBe(0)
+        })
+    })
+    test('Returns a 404 when comment_id is not found', ()=>{
```

Comment:
We should respond with 404 if the id is of the right type (a number) but not present in the database i.e. /api/comments/99999
While we should respond with a 400 if the id is not of the right type i.e. /api/comments/embarrassingRagePost 🙂